src/system.c

```diff
@@ -1063,6 +1063,12 @@ void InitSystem (
 #ifdef SYS_IS_64_BIT
     SyStorMin = 128 * 1024L;
     SyStorMax = 2048*1024L;          /* This is in kB! */
+#if defined(HAVE_SYSCONF)
+#if defined(_SC_PAGESIZE) && defined(_SC_PHYS_PAGES)
```

Comment:
Not a nitpick, genuine question: is there a reason why these `#if`s are split across two lines, not not just one with another `&&`?

Reply:
Just the line will be very long, and it looks weird when preprocessor lines are split